test/jdk/java/awt/Graphics/LCDTextAndGraphicsState.java

```diff
@@ -21,26 +21,46 @@
  * questions.
  */
 
-/**
+/*
  * @test
  * @bug 6576507
  * @summary Both lines of text should be readable
- * @run main/manual=yesno LCDTextAndGraphicsState
+ * @run main/manual LCDTextAndGraphicsState
  */
-import java.awt.*;
-import java.awt.geom.*;
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.GridLayout;
+import java.awt.RenderingHints;
+import java.awt.AlphaComposite;
+import java.awt.GradientPaint;
+import java.awt.Shape;
+import java.awt.Panel;
+
+import java.awt.event.ActionEvent;
+import java.awt.geom.RoundRectangle2D;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
```

Comment:
I'd also suggest sorting imports alphabetically. Your IDE could do this for you with its Optimize Imports command.

Reply:
Done